submissions/AymanIsmail/assignment8.js

```diff
@@ -0,0 +1,44 @@
+// Task 1
+const editor = {
+  title: "my first blog",
+  getUpperTitle() {
+    return this.title.toUpperCase();
+  },
+};
+
+const getTitle = editor.getUpperTitle.bind(editor);
+
+console.log(getTitle());
+//"MY FIRST BLOG"
+
+// Task 2
+const formHandler = {
+  value: "initial",
+  onChange(newValue) {
+    this.value = newValue;
+  },
+};
+
+function simulateInputChange(callback) {
+  callback("updated");
+}
+
+simulateInputChange(formHandler.onChange);
+
+console.log(formHandler.value); //
+"updated"
```

Comment:
In task 2 u had to use bind or arrow function, and don't use none of them. also u need to explain why u use it
Actually, I prefer from u to use both of them